generator/mapping.json

```diff
@@ -0,0 +1,140 @@
+{
```

Comment:
There isn't documentation for the effect of each key-value pair that appears in this file, ex. params_additional and the array value it has. This would be useful information to add to the README of how each override rule works, as we may need to make changes to this file in future releases.

There should be a clearer distinction as to whether properties under a struct/enum/message are attributes of that class, or whether they are doing some modification to that class. Ex. under RegisterAppInterfaceRequest there is appID and fullAppID which are properties of RegisterAppInterfaceRequest, but there is also the params_additional property on the same level which wouldn't actually be a property of RegisterAppInterfaceRequest. Another example is PutFileRequest, where syncFileName is a property of the class, but script is not and it performs another effect.

Reply:
The documentation added into readme b30da9358e809a1283c961c5937b641e965927ea